velox/exec/Task.cpp

```diff
@@ -1827,6 +1825,38 @@ void Task::TaskCompletionNotifier::notify() {
   }
 }
 
+void Task::createExchangeClient(
+    int32_t pipelineId,
+    const core::PlanNodeId& planNodeId) {
+  std::unique_lock<std::mutex> l(mutex_);
+  VELOX_CHECK_NULL(
+      getExchangeClientLocked(pipelineId),
+      "Exchange client has been created at pipeline: {} for planNode: {}",
+      pipelineId,
+      planNodeId);
+  VELOX_CHECK_NULL(getExchangeClientLocked(planNodeId));
```

Comment:
consistency: the previous check has a detailed error message, but this check doesn't.